src/main/java/org/spongepowered/common/service/pagination/ActivePagination.java

```diff
@@ -39,42 +40,36 @@
 import java.lang.ref.WeakReference;
 import java.util.ArrayList;
 import java.util.List;
+import java.util.Map;
 import java.util.UUID;
 
 /**
  * Holds logic for an active pagination that is occurring.
  */
-abstract class ActivePagination {
+class ActivePagination {
```

Comment:
Nope.

The point of having separate pagination types is so that plugins wanting to send really long lists to players don't have to store everything in memory. not getting rid of that

additionally, the changes to next/previous page texts get rid of nice unicode arrows are not needed